source/common/upstream/cluster_manager_impl.cc

```diff
@@ -313,12 +313,21 @@ void ClusterManagerImpl::onClusterInit(Cluster& cluster) {
   // Now setup for cross-thread updates.
   cluster.prioritySet().addMemberUpdateCb(
```

Comment:
Missed this on my last pass: this callback only triggers when hosts are added or removed. For example, a change to host weights would not trigger this callback, instead only triggering callbacks added with `addPriorityUpdateCb` for the appropriate priority. Not sure if this is intended or not, just pointing it out.